api/filter/accesslog.proto

```diff
@@ -95,16 +258,63 @@ message FileAccessLog {
   string format = 2;
 }
 
+// Configuration for the built-in "envoy.grpc_access_log" access log type.
+message AccessLogServiceConfig {
+  // The friendly name of the access log to be returned in StreamAccessLogsMessage.Prefix. This
+  // allows the access log server to differentiate between different access logs coming from the
+  // same Envoy.
+  string log_name = 1;
```

Comment:
Shouldn't there be a cluster name in this config?

Reply:
Oops yes. I would have realized this when implementing! Thanks.